### DISCUSSION TITLE
Mesh problem?

### DISCUSSION THREAD

**Discussion by Joseph-0123** [idaholab/moose#23403] [Q&A General] [answered]

**Comment by Joseph-0123** [comment]
Hello @GiudGiud , do you have any idea? Thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

The mesh must be different in some way.
Could you please upload the mesh in a github repo?
Downloading zip files is frowned upon

Guillaume

**Reply by GiudGiud** [reply]
I think the issue is that the mesh is being detected as a 3D mesh when it s really 1D.
I m not sure what you can do in gmsh to make it more 1D.
Maybe there's a number you can change in the .msh file??
Otherwise, more hacky approaches:
Maybe make X and Y equal to 0. Maybe use only the X axis instead of the Z axis for the 1D mesh.

**Comment by Joseph-0123** [comment]


**Reply by GiudGiud** [reply]
You can transfer from a 1D mesh to a 3D model with no problem. 
You dont need both meshes to be 3D.

**Comment by Joseph-0123** [comment]


**Reply by GiudGiud** [reply]
For transfers? Any transfer will behave as expected. ShapeEvaluation or NearestNode are likely the best choices for it

**Reply by GiudGiud** [reply]
For making the mesh 1D I have no idea. You should try the options I offered. You can always translate the mesh in MOOSE using a TransformGenerator
